Source/WebCore/rendering/svg/legacy/LegacyRenderSVGPath.cpp

```diff
@@ -63,6 +63,17 @@ void LegacyRenderSVGPath::updateShapeFromElement()
         m_shapeType = ShapeType::Line;
     else
         m_shapeType = ShapeType::Path;
+
+    // FIXME: This should not exist. However, currently SVG is relying on ordering of calculation of strokeBoundingBox via layout() function
+    // for recursive SVGs (markers are pointing to each other recursively). If we move to SVG2 compuation, we no longer need this since SVG2 strokeBoundingBox
+    // does not include markers rect (so we do not need to have this in LSBE). Right now, this exists only for RepaintRectCalculation::Accurate, and it should be removed once
```

Comment:
This and the commit message makes me slightly confused, because (per my comment on the top level PR) "stroke bounding box" is not a concept that exists in SVG 1. The fact that our LegacyRenderSVG* classes talk about stroke bounding boxes is really just a poor name for repaint rects, I think.